pkg/doc/verifiable/presentation.go

```diff
@@ -402,6 +403,14 @@ func WithPresJSONLDDocumentLoader(documentLoader jsonld.DocumentLoader) Presenta
 	}
 }
 
+// WithDisabledJSONLDChecks disables JSON-LD checks for VP parsing.
+// By default, JSON-LD checks are enabled.
+func WithDisabledJSONLDChecks() PresentationOpt {
```

Comment:
Please specify in comments atleast, what you mean by JSON LD checks, is it JSON LD Schema Validation?